Añadir, editar y eliminar usuarios desde admin [Frontend]

IsmaelRuizJurado opened the pull request:
#4 
Funcionalidad base añadida con validaciones básicas.
Cosas que sé que faltan (posiblemente implementarlas en un futuro cercano):
-Que las acciones sean realizables por un admin (pues no contaba con la funcionalidad de login)
-Restricción sobre fecha para la mayoría de edad de los usuarios
-Refactorización general
-Cambiar los estilos para que sean acordes al resto de la aplicación

Para probar la funcionalidad:
En front, usar la ruta users. Clicar sobre el nombre de un usuario para probar las funcionalidades de actualizar y borrar.
Para añadir un nuevo usuario usar la ruta add.

IsmaelRuizJurado committed:
feat: Crud usuarios por admin iniciado #4. Queda gran parte del frontend por implementar así como las pruebas

IsmaelRuizJurado committed:
feat: añadir, editar y borrar usuarios desde admin

Funcionalidad base implementada con validaciones básicas. Queda por añadir que solo un admin pueda hacer estas operaciones.

IsmaelRuizJurado committed:
Merge branch 'epic/user-management/1' into feat/add-edit-delete-users-by-admin/4

IsmaelRuizJurado commented:
Conflictos solucionados.

JoseJavierAlcobendas reviewed (changes requested):
Se que sabes que te falta refactorizar y los estilos sin embargo lo suyo sería que lo hicieses en esta pull request y en cuanto a la funcionalidad de añadir un user porque no pones un boton en el listado de usuarios que te lleve a la url /add.
Por lo demas muy buen trabajo!!

virolialo reviewed (changes requested):
Las pantallas funcionan bien, aunque estoy con Jose, hace falta refactorizar código.

También habría que crear una carpeta raíz para los componentes de usuario, y ahí meter todas las que se han ido creado relacionados con este, pero esto se podría solucionar una vez se integre todo en la épica de la gestión de usuario.

Buen trabajo Ismael 👍

IsmaelRuizJurado committed:
Feat: Refactorización según el feedback #4

Realizadas pequeñas refactorizaciones del código de los componentes del frontend y añadido el botón en la lista de usuarios para añadir un usuario

IsmaelRuizJurado committed:
Merge branch 'feat/add-edit-delete-users-by-admin/4' of https://github.com/Aparking/AparKing_Frontend into feat/add-edit-delete-users-by-admin/4

IsmaelRuizJurado commented:
He añadido el botón mencionado y he realizado una pequeña refactorización del código de los componentes. En cuanto a los estilos he decidido dejarlos con los predeterminados de ionic para más tarde cambiarlos según el resto de implementaciones.

laurolmer reviewed (approved):
lo veo bien

perealberto reviewed (approved):
Lo veo bien @IsmaelRuizJurado, te acepto los cambios 😺

virolialo reviewed (approved):
Todo en orden !! Me alegra que hayas seguido en consejo :)
